### PR TITLE
Feature/add institutions

### PR DESCRIPTION
**Pull request by skriyaz95**

### CODE REVIEW

File: src/components/home/PanelResult.vue [hunk 1]
```diff
@@ -53,6 +130,17 @@
               </v-toolbar-title>
             </v-toolbar>
           </template>
+          <template v-slot:[`item.institution`]="{ item }">
+            <v-tooltip bottom>
+              <template v-slot:activator="{ on }">
```
Comment (cmonnom) [inline src/components/home/PanelResult.vue:135]: [vue/no-unused-vars]
'on' is defined but never used.
You're missing the v-on="on" in the element doing the activation of the tooltip

File: src/components/home/PanelResult.vue [hunk 2]
```diff
@@ -1,19 +1,88 @@
...
+            <v-list-item-content>
+              <v-list-item-title><a :href="linkTo(institution.email, 'email')">{{ institution.email }}</a></v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+          <v-list-item>
+            <v-list-item-icon>
+              <v-icon>mdi-earth</v-icon>
+            </v-list-item-icon>
+            <v-list-item-content>
+              <v-list-item-title><a :href="institution.website" target="_blank">Go to Website</a></v-list-item-title>
```
Comment (cmonnom) [inline src/components/home/PanelResult.vue]: replace with $t()

File: src/components/home/PanelResult.vue [hunk 3]
```diff
@@ -53,6 +130,17 @@
               </v-toolbar-title>
             </v-toolbar>
           </template>
+          <template v-slot:[`item.institution`]="{ item }">
+            <v-tooltip bottom>
+              <template v-slot:activator="{ on }">
+                <span class="pointer" v-if="Object.keys(item.institution).length > 0" @click.stop="openInstitutionDetails(item.institution)">
```
Comment (cmonnom) [inline src/components/home/PanelResult.vue]: use v-btn with href instead. Remove class .pointer

replace v-if="..." with a method instead to make it easier to debug

File: src/components/home/PanelResult.vue [hunk 4]
```diff
@@ -147,6 +251,7 @@ export default Vue.extend({
     ...mapGetters({
       userGenes: 'getUserGenes',
       userGenesInPanels: 'getUserGenesInPanels',
+      institutionsMap: 'getInstitutionsMap'
```
Comment (cmonnom) [inline src/components/home/PanelResult.vue]: Rename (and update references) to institutionMap: 'getInstitutionMap' (no need for the 's')

**Review by cmonnom — CHANGES_REQUESTED** [review]
It's working. A few minor things to change. Good job

File: src/components/home/PanelResult.vue (hunk 1 above)
Comment (skriyaz95) [reply src/components/home/PanelResult.vue:135]: updated

File: src/components/home/PanelResult.vue (hunk 2 above)
Comment (skriyaz95) [reply src/components/home/PanelResult.vue]: updated

File: src/components/home/PanelResult.vue (hunk 3 above)
Comment (skriyaz95) [reply src/components/home/PanelResult.vue]: updated

File: src/components/home/PanelResult.vue (hunk 4 above)
Comment (skriyaz95) [reply src/components/home/PanelResult.vue]: updated

File: src/components/home/PanelResult.vue (hunk 3 above)
Comment (cmonnom) [reply src/components/home/PanelResult.vue]: Don't forget to change the span into a v-btn + remove pointer class. I was confused about the href, the @click does the trick.

Comment (skriyaz95) [reply src/components/home/PanelResult.vue]: done

**Pull request merged**